### DISCUSSION TITLE
transparent-background Discussions

### DISCUSSION THREAD

**Discussion by plemeri** [plemeri/transparent-background#28] [Ideas]
There's a problem with Google Drive URL and md5 checksum. The related issue is https://github.com/plemeri/transparent-background/issues/19 and I want some opinions to solve this issue. Please feel free to suggest any idea.

Thanks.

**Comment by Satyaexp** [comment]
hey thanks for your earlier reply, it did sort the issues pertaining to those images, but there are now a few more that are giving issues....  what do you think is the issue... attached images for your reference.  https://drive.google.com/file/d/17r7bk_lrnYBMkd1b3O21O0szTzQ9eEFp/view?usp=sharing
